lib/activerecord-import/adapters/mysql_adapter.rb

```diff
@@ -84,13 +84,13 @@ def add_column_for_on_duplicate_key_update( column, options = {} ) # :nodoc:
 
   # Returns a generated ON DUPLICATE KEY UPDATE statement given the passed
   # in +args+.
-  def sql_for_on_duplicate_key_update( table_name, *args ) # :nodoc:
+  def sql_for_on_duplicate_key_update( table_name, locking_column, *args ) # :nodoc:
     sql = ' ON DUPLICATE KEY UPDATE '
     arg = args.first
     if arg.is_a?( Array )
```

Comment:
`locking_column = args.last`

Reply:
When locking_column parameters depended in order, I felt it was difficult to change in the future so I did this.